packages/vite/package.json

```diff
@@ -18,7 +18,7 @@
     "!/src/**/__tests__/"
   ],
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=12.2.0"
```

Comment:
It's because `createRequire` is only supported in Node.js >= 12.2.0.

There's a `createRequireFromPath` function that does exactly the same job in previous versions. So this line of change is not a must.

But I'm wondering, do we really need to support those old versions? They might have serious bugs that make other parts of Vite not work correctly.

Reply:
Do you think we need to merge this in a minor because of this change?

Reply:
Yeah, it needs to be a minor, especially with this new feature: [`9d1c05f` (#4279)](https://github.com/vitejs/vite/pull/4279/commits/9d1c05fda32cdf60efb6af95f0098e3f5c54ad4f)